modules/globebrowsing/globebrowsingmodule.cpp

```diff
@@ -258,6 +259,8 @@ void GlobeBrowsingModule::internalInitialize(const ghoul::Dictionary& dict) {
             static_cast<size_t>(CpuCap.installedMainMemory() * 0.25 * 1024 * 1024)
         );
         addPropertySubOwner(GdalWrapper::ref());
+
+        CPLSetConfigOption("GDAL_HTTP_TIMEOUT", "15");
```

Comment:
Would be better to move this to the GDALWrapper class where we already have:

`    CPLSetConfigOption("GDAL_HTTP_TIMEOUT", "3"); // 3 seconds`